ProcessOptimizer/plots.py

```diff
@@ -1,2382 +1,2493 @@
+
+    Parameters
+    ----------
+    * `result` [`OptimizeResult`]
+        The result for which to create the scatter plot matrix.
+
+    * `levels` [int, default=10]
+        Number of levels to draw on the contour plot, passed directly to
+        `plt.contour()`.
+
+    * `oversampling` [int, default=10]
+        How many sample to have per 2d pixel on average.
+
+    * `graph_width` [int, default=20]
+        Number of points at which to evaluate the partial dependence along
+        each dimension.
+
+    * `size` [float, default=2]
+        Height (in inches) of each facet.
+
+    * `zscale` [str, default="linear"]
+        Scale to use for the z axis of the contour plots. Either 'linear' or 'log'.
+
+    * `dimensions` [list of str, default=None]
+        Labels of the dimension variables. `None` defaults to
+        `space.dimensions[i].name`, or if also `None` to `['X_0', 'X_1', ..]`.
+
+    * `usepartialdependence` [bool, default=True]
+        Whether to use partial dependence or not when calculating dependence. If false,
+        plot_objective will parse values to the dependence function, defined by the pars
+        argument.
+
+    * `pars` [str or list of floats, default="result"]
+        Defines the values for the red points in the plots, and if partialdependence is
+        false, this argument also defines values for all other parameters when
+        calculating dependence.
+        Valid strings:
+            "result" - Use best observed parameters
+            "expected_minimum" - Parameters that gives the best minimum. Calculated
+                using scipy's minimize method. This method currently does not work with
+                categorical values.
+            "expected_minimum_random" - Parameters that gives the best minimum when
+            using naive random sampling. Works with categorical values.
+            '[x[0], x[1], ..., x[n]] - Parameter to show depence from a given x.
+
+    * `expected_minimum_samples` [float, default = None]
+        Determines how many points should be evaluated to find the minimum when using
+        'expected_minimum' or 'expected_minimum_random'.
+
+    * `title` [str, default=None]
+        String to use as title of the figure.
+
+    * `show_confidence` [bool, default=False]
+        Whether or not to show a credible range around the mean estimate on the 1d-plots
+        in the diagonal. The credible range is given as 1.96 times the std in the point.
+
+    * `plot_options` [dict, default=None]
+        Dictionary with the options to use when plotting. If None, the defaults are used
+        for all values.
+        Possible keys:
+        * "interpolation" [string, default=""]: The interplotation visaulization to use for the 2D
+            plots. For possible values, see
+            https://matplotlib.org/stable/gallery/images_contours_and_fields/interpolation_methods.html
+            If empty string, a contour pot is shown. Note that contour plots are not
```

Comment:
"plot" not "pot". It should be made clear that contour plots are not compatible with showing uncertainty _in the 2D plots_.